lib/crypto/crypto_internal.h

```diff
@@ -3,7 +3,6 @@
 
 #include <stdint.h>
 
-#include <openssl/aes.h>
```

Comment:
This header file existed long before tarsnap 1.0.12; that's just when it was moved to its current location.

In tarsnap 0.1.5 it contained

```
struct crypto_aes_key {
        AES_KEY key;                    /* Expanded key. */
        uint64_t nonce;                 /* Nonce, used only for encryption. */
        uint8_t key_encrypted[256];     /* AES key encrypted with encr_pub. */
};
```

which is why it needed `<openssl/aes.h>`.

Can you change the commit message to just say that it was needed in previous versions but is no longer necessary?